src/main/java6/net/finmath/time/businessdaycalendar/BusinessdayCalendarExcludingTARGETHolidays.java

```diff
@@ -45,7 +44,9 @@ public boolean isBusinessday(LocalDate date) {
 		LocalDate datePlus2 = date.plusDays(2);
 		LocalDate dateBefore = date.minusDays(1);
 
-		return	(baseCalendar == null || baseCalendar.isBusinessday(date))
+		BusinessdayCalendarInterface weekdayCalendar = new BusinessdayCalendarExcludingWeekends();
```

Comment:
Construction of a new object in every call of isBusinessday may be a performance issue.